tests/pages/components/user-sidebar.js

```diff
@@ -0,0 +1,15 @@
+import { isVisible, text, isHidden, property } from 'ember-cli-page-object';
+
+export default {
+  scope : '.user-sidebar',
+
+  userSidebarCount    : isVisible(),
+  name                : text('h2 .name'),
+  username            : text('h2 .username'),
+  twitterHandle       : text('li.twitter'),
+  website             : text('li.website'),
+  twitterHandleHidden : isHidden('li.twitter'),
+  websiteHidden       : isHidden('li.website'),
+  twitterLink         : property('href', 'li.twitter a'),
+  websiteLink         : property('href', 'li.website a')
```

Comment:
alright cool. I was actually supposed to introduce that with https://github.com/code-corps/code-corps-ember/pull/653 which got stale and I'll do another PR, but I thought I should do it once the remaining of my PRs are merged.